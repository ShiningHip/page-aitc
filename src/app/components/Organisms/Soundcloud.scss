.soundcloud {
  background: rgba(0, 0, 0, 0.7);
  min-height: 300px;
  position: relative;
  display: block;
  padding-top: 30px;
  z-index: 20;
}

.soundcloud_container {
  width: 30%;
  height: 300px;
  position: relative;
  text-align: right;
}

@media screen and (min-width: 1280px) {
  .soundcloud_container {
    min-width: 1000px;
    right: 0;
    left: 0;
    margin: auto;
  }
}

@media screen and (max-width: 1280px) {
  .soundcloud_container {
    width: 100%;
  }
}
