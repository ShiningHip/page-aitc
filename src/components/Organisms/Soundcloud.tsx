import React from 'react'
import style from './Soundcloud.scss'

const Soundcloud: React.FC = () => (
  <>
    <style jsx>{style}</style>
    <div className="soundcloud">
      <div className="soundcloud_container">
        <iframe
          width="100%"
          height="300"
          scrolling="no"
          allow="autoplay"
          src="https://w.soundcloud.com/player/?url=https%3A//api.soundcloud.com/tracks/549842910&color=%23ff5500&auto_play=false&hide_related=false&show_comments=true&show_user=true&show_reposts=false&show_teaser=true&visual=true"
        ></iframe>
      </div>
    </div>
  </>
)

export default Soundcloud
