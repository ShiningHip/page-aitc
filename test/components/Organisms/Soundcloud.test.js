import Soundcloud from 'components/Organisms/Soundcloud'
import { configure, shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
configure({ adapter: new Adapter() })

describe('Soundcloud test', () => {
  it('表示が正しいこと', () => {
    const tree = shallow(<Soundcloud />).toJSON
    expect(tree).toMatchSnapshot()
  })
})
