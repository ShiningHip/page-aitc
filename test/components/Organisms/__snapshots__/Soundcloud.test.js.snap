// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Soundcloud test 表示が正しいこと 1`] = `undefined`;
